### ISSUE TITLE
sampling is different between diffusers and sat ?

### ISSUE THREAD

**Issue by lith0613** [THUDM/CogVideo#482]
Hello,
I am currently encountering an issue where the effect becomes noticeably blurry after using conversion code to transform a SAT training model into diffusers. I have pinpointed that the model parameters are aligned, but there is a clear discrepancy in the output of the scheduling part. Specifically, the output after executing scheduler.step in the diffusers code does not align at all with the code in the SAT version’s sampling.py file.

**Comment by zRzRzRzRzRzRzR** [comment]
Regarding the 2B version, I think there might be some differences. Perhaps @a-r-r-o-w could help me with an explanation of this part?
